NSpecSpecs/describe_RunningSpecs/Exceptions/describe_expected_exception_in_act.cs

```diff
@@ -53,7 +53,89 @@ void method_level_context()
 
                 context["when exception thrown from act"] = () =>
                 {
-                    actAsync = async () => await Task.Run(() => { throw new InvalidOperationException("Testing"); });
```

Comment:
Wouldn't it be better to leave `async`/`await`, as with all other async test cases?